app/utils/utils.js

```diff
@@ -26,6 +32,38 @@ const getPrivateKeyAddress = (network, privateKey) => {
   return privateKey.address;
 };
 
+const mnemonicToPrivateKey = (mnemonic) => {
+	const seed = bip39.mnemonicToSeed(mnemonic)
+
+	const master = bip32.fromSeed(seed)
+  const child = master.derivePath(`m/44'/5757'/0'/0/0`)
+
+  const ecPair = ECPair.fromPrivateKey(child.privateKey)
+
+  return ecPairToHexString(ecPair)
+}
+
+const mnemonicToStxAddress = (mnemonic) => {
+  const seed = bip39.mnemonicToSeed(mnemonic)
+
+	const master = bip32.fromSeed(seed)
+	const child = master.derivePath(`m/44'/5757'/0'/0/0`)
+
```

Comment:
Should this just be:

```javascript
b58ToC32(ecPairToAddress(ECPair.fromPrivateKey(child.privateKey)))
```

That way, we don't duplicate the hash computation, address generation code.